setup.py

```diff
@@ -21,4 +21,8 @@
     description='Cibyl is tool that models ci system by'
     'creating in-memory representation of CI system entities',
     version='0.0.1',
+    entry_points={
+        'console_scripts': ['cibyl = cibyl.cli.main:main',
+                            'ci = cibyl.cli.main:main']
```

Comment:
Why we need 'ci' executable?

Reply:
heh good question :)
I admit I thought it could be nice to have shorter alias for cibyl using simply 'ci'. But yeah, it doesn't really make sense, so if you think we should remove it, I'm fine with that :)

Reply:
Great! +1 for ci.